### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ghost**
This pr includes minor grammatical corrections.

### CODE REVIEW

File: part1/example1/README.md [hunk 1]
```diff
@@ -81,6 +81,6 @@ For example in the diagram above I had
 require('./styles.css')
 ```
 
-If I include [the css-loader](https://github.com/webpack/css-loader) in my webpack config, this is not only perfectly valid, but also will actually apply the CSS to my page.
+Including the [the css-loader](https://github.com/webpack/css-loader) in my webpack config is not only perfectly valid, but it will also actually apply the CSS to my page.
```
Comment (AriaFallah) [inline part1/example1/README.md]: This changes the meaning of the sentence. Also it says `the the` now.

**Comment by AriaFallah** [comment]
The first comma removal was a good catch, but your changes to the sentence about the css-loader change the meaning of the sentence. If you limit it to just that first comma, I'll merge this.

**Comment by ghost** [comment]
I reversed the second change. Thanks!

**Comment by AriaFallah** [comment]
Perfect thanks!

**Pull request merged**